packages/mdc-top-app-bar/README.md

```diff
@@ -95,6 +116,10 @@ Method Signature | Description
 `registerNavigationIconInteractionHandler(evtType: string, handler: EventListener) => void` | Registers an event listener on the native navigation icon element for a given event.
 `deregisterNavigationIconInteractionHandler(evtType: string, handler: EventListener) => void` | Deregisters an event listener on the native navigation icon element for a given event.
 `notifyNavigationIconClicked() => void` | Emits a custom event `MDCTopAppBar:nav` when the navigation icon is clicked. 
+`registerScrollHandler(handler) => void` | Registers a handler to be called when user scrolls. Our default implementation adds the handler as a listener to the window's `scroll` event.
+`deregisterScrollHandler(handler) => void` | Unregisters a handler to be called when user scrolls. Our default implementation removes the handler as a listener to the window's `scroll` event.
+`getViewportScrollY() => number` | Gets the number of pixels that the content of body is scrolled upward.
+`totalActionIcons() => number` | Gets the number of action items on the right side of the top app bar.
```

Comment:
We refer to action items in prose but the API uses the word Icons? Do we ever expect any action items that aren't icons?

Reply:
Spec refers to them as `Action Items` but never uses anything other than icons. I'll change all of them to Items for consistency.